shared/ssl.sh

```diff
@@ -100,9 +100,11 @@ openssl genrsa -out "${PRIVKEYFILE}" 4096
 openssl rsa -in "${PRIVKEYFILE}" -pubout -out "${PUBKEYFILE}"
 openssl req -new -key "${PRIVKEYFILE}" -out "${CSRFILE}" -subj "${CERTSUBJECT}"
 
-### Submit CSR
-curl ${CURLFLAGS} -X PUT -H "Content-Type: text/plain" \
-    --data-binary "@${CSRFILE}" "${CA}/certificate_request/${CERTNAME}"
+### Submit CSR and fail if one already exists on the CA
+output="$(curl ${CURLFLAGS} -X PUT -H "Content-Type: text/plain" \
+               --data-binary "@${CSRFILE}" "${CA}/certificate_request/${CERTNAME}")"
+already_exists="${CERTNAME} already has a requested certificate; ignoring certificate request"
+[ "${output}" = "${already_exists}" ] && error "unsigned CSR already exists on CA"
```

Comment:
This change is for https://trello.com/c/ObwIZSwo/119-sslsh-improvement-when-cert-exists-already

...which was meant to say "when CSR already exists"